pkg/utils/apiutil/apiutil.go

```diff
@@ -50,7 +50,7 @@ const (
 	// PDRedirectorHeader is used to mark which PD redirected this request.
 	PDRedirectorHeader = "PD-Redirector"
 	// PDAllowFollowerHandleHeader is used to mark whether this request is allowed to be handled by the follower PD.
-	PDAllowFollowerHandleHeader = "PD-Allow-follower-handle"
+	PDAllowFollowerHandleHeader = "PD-Allow-follower-handle" // #nosec G101
```

Comment:
I am confused why we have to add this comment :(

Reply:
It seems like a false positive error:

```shell
pkg/utils/apiutil/apiutil.go:53:2: G101: Potential hardcoded credentials (gosec)
	PDAllowFollowerHandleHeader = "PD-Allow-follower-handle"
	^
```